### PR TITLE
Multiple counters issue#40

### PR DESCRIPTION
**Pull request by nelsonic**
Working on adding Multiple Counters with tests for #40 ...
will _assign_ when ready for review. 👍

### CODE REVIEW

**Comment by w0rm** [comment]
@nelsonic I’m not the author of this illustration, please provide correct credits. https://twitter.com/01k/status/986528602635358208?s=20

**Comment by nelsonic** [comment]
@w0rm thank you for taking time to _comment_, 🎉 
I have added the `(CC BY 4.0)` attribution below the image. ✅ 
_Fantastic_ drawing BTW! ❤️ 
and great work on the Game tutorial: https://github.com/w0rm/creating-a-fun-game-with-elm 👾 
and presentation: https://unsoundscapes.com/slides/2016-10-13-creating-a-fun-game-with-elm 💎

File: README.md [hunk 1]
```diff
@@ -105,17 +105,36 @@ If you have **_any_ questions**, ***please ask***: <br />
...
 
 ![elm-muv-architecture-diagram](https://cloud.githubusercontent.com/assets/194400/25773775/b6a4b850-327b-11e7-9857-79b6972b49c3.png)
 
-Don't worry if you don't understand this diagram (_yet_),
+<br />
+If you're not into flow diagrams, don't worry, there not everyone is, <br />
+a _much_ more "user friendly" _explanation_
+of **The Elm Architecture** ("TEA") <br />
+is
+[**Andrey Kuzmin**'s](https://github.com/w0rm) _fantastic_
```
Comment (w0rm) [inline README.md]: This is not my illustration and should not mention my name, Kolja Wilcke https://twitter.com/01k made it.

**Comment by w0rm** [comment]
@nelsonic thanks for correcting the attribution. All the credits should really go to Kolja.

File: README.md (hunk 1 above)
Comment (nelsonic) [reply README.md]: @w0rm got it. thanks for clarifying. fixed. 👍

**Pull request merged**